### ISSUE TITLE
GUI doesn't launch using desktop file

### ISSUE THREAD

**Issue by ashucg** [flathub/org.gnome.Hamster#2]
As mentioned in the title, GUI does not launch when you use the `.desktop` file (menu entry), `flatpak run org.gnome.Hamster` works fine.

**Comment by rabin-io** [comment]
@ashucg can you please try the fix mention above, and comment the `DBusActivatable=true` line in the file 

```
/var/lib/flatpak/exports/share/applications/org.gnome.Hamster.desktop
```

**Comment by ashucg** [comment]
@rabin-io yes, I can confirm that commenting out the `DBUSActivatable=true` line fixes the issue.

**Comment by rabin-io** [comment]
I will need some time to investigate this, as D-Bus activation should be supported on flatpak as well

**Comment by bbhtt** [comment]
I think this is because the desktop file in the flatpak is called `org.gnome.Hamster` and not org.gnome.Hamster.GUI` and upstream uses the latter as the id.

Can you rename it to that and see if it works?

ON GNOME:

```
 cp {/var/lib/, ~/.local/share/}flatpak/exports/share/applications/org.gnome.Hamster.desktop ~/.local/share/applications

mv ~/.local/share/applications/org.gnome.Hamster.desktop ~/.local/share/applications/org.gnome.Hamster.GUI.desktop
```

Now log out and log back in and try to launch it. Seems to work here.

If this is the case, I think the solution is to remove that part from the upstream desktop file since we can't have two desktop files or a single one with `org.gnome.Hamster.GUI`.

Otherwise patching the upstream service files could be an idea. I haven't looked into it.

**Comment by magicmyth** [comment]
I've just run into this issue myself and editing a copy of the .desktop file to have the `org.gnome.Hamster.GUI.desktop` indeed solved the issue for me. Note that when I launch the original desktop file hamster-service does correctly start in the background regardless.

This is under KDE Plasma 5.27. As a side note I get no icon in the indicator tray which was working with an old version I built from June 16th 2021 (I can't find the exact commit it was based on).

Just in case its useful I was led here from the application issue projecthamster/hamster#692.
